modes/ement/evil-collection-ement.el

```diff
@@ -0,0 +1,72 @@
+;;; evil-collection-ement.el --- Evil integration for Ement.el  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2023  Nicholas Vollmer
+
+;; Author:  Nicholas Vollmer
+;; Keywords: convenience
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;;
+
+;;; Code:
+(require 'evil-collection)
+(require 'ement nil t)
+(require 'ement-room-list nil t)
+
+(defgroup evil-collection-ement nil
+  "Evil bindings for Ement.el."
+  :group 'evil-collection)
+
+(defcustom evil-collection-ement-want-auto-retro t
```

Comment:
Given that this is defined as an advice, this is a bit invasive to enable by default. I'd suggest either just defining/binding wrapper commands, or disabling this by default.

Reply:
Fair point. I'll disable by default.

Reply:
Implemented via hooks now. Disabled by default.